05_service_configuration/050_caching_dns_server/051_caching_dns_server.md

```diff
@@ -0,0 +1,28 @@
+# Configure a Caching DNS Server
+This section is based on this [cloudflare article](https://www.cloudflare.com/learning/dns/what-is-recursive-dns/) and this [LinuxTeck guide](https://www.linuxteck.com/how-to-setup-caching-dns-server-in-centos-rhel-7-6/).
```

Comment:
I think you dont need the word "this" to reference your sources. Just mention that this section is based on an cloudflare article and a how to guide from linuxteck as well as add the corresponding links. Sounds more academic :)